docs-site/src/markdown-pages/index.md

```diff
@@ -0,0 +1,113 @@
+---
+key: index
+---
+
+## Overview
+
+EOP is an initiative led by Greater Wellington Regional Council building on the
+experience of the Environment Canterbury Water Data programme to build a
+solution which will allow all regional councils to maximise the value they get
+from their environmental monitoring efforts and provide clear, consistent, and
+user-centric information to users of environmental information.
+
+Regional councils have many systems for collecting and managing environmental
+data. These systems are often siloed, legacy, and not integrated. This makes
+integrating the data to derive information a manual task of combining the data
+from multiple systems, which is slow, error-prone, and often this is done in a
+way that isn't consistent within a single council or across councils. This
+effort to integrate the data limits the type of analysis that people will even
+try to achieve with the data. This leads to the value of the data not being
+fully realised.
+
+Much work has been done in councils by people without a software or data
+engineering background on the best efforts' basis. This has resulted in
+solutions that deliver lots of value but lack the maintainability, scalability
+and user-centric approach that would come from a software engineering team.
+
+EOP aims to address these issues by building a solution which focuses on:
+
+_Outcomes_
+
+The unit of delivery in EOP is an outcome, information provided to a user in a
+way which is appropriate to their needs. For each outcome, the minimum technical
+architecture will be built to deliver that outcome. With future outcomes
+building and evolving, the architecture to support the new outcomes.
+
+_Multi Tenanted_
+
+From the ground up EOP is being built to support multiple councils. This will
+enable the effort to deliver an outcome by one council to be delivered to all
+councils for an incremental cost. This means in general, EOP will be agnostic of
+council internal systems with each council providing data to EOP in a
+standardised format. However, for some common systems used in a majority of
+councils, EOP can provide standardised adapters to allow the data to be
+
+_Modern Development Practices_
+
+We are using modern deliver practices ... why do you care? It supports long term
+maintainability and reduces the risk of the platform becoming legacy.
```

Comment:
There's a switch to a different perspective here ("We"). Also, is it worth a very brief sentence explaining what some of these are? E.g.

> The EOP platform will be developed using modern development and delivery practices to reduce the risk of the platform becoming legacy, and help ensure future development and operation and is sustainable. This includes test coverage, infrastructure automation and a _devops_ approach, and selecting fit-for-purpose tools, infrastructure and standards.